pyls/plugins/jedi_completion.py

```diff
@@ -103,16 +112,17 @@ def _format_completion(d, include_params=True):
     if include_params and hasattr(d, 'params') and d.params:
         positional_args = [param for param in d.params if '=' not in param.description]
 
-        # For completions with params, we can generate a snippet instead
-        completion['insertTextFormat'] = lsp.InsertTextFormat.Snippet
-        snippet = d.name + '('
-        for i, param in enumerate(positional_args):
-            name = param.name if param.name != '/' else '\\/'
-            snippet += '${%s:%s}' % (i + 1, name)
-            if i < len(positional_args) - 1:
-                snippet += ', '
-        snippet += ')$0'
-        completion['insertText'] = snippet
+        if len(positional_args) > 1:
```

Comment:
I think we should insert `()` and put the cursor in the middle of them, in case `positional_args == 1`.

Reply:
For `positional_args == 0` we need to put the cursor to the right of `()`

Reply:
For `positional_args == 0` we need to put the cursor to the right of `()`